cmd/network/verification.go

```diff
@@ -650,7 +651,7 @@ func (e *EgressVerification) getCaBundleFromManagementCluster(ctx context.Contex
 		return "", err
 	}
 
-	mc, err := utils.GetManagementCluster(e.cluster.ID())
+	mc, _, err := utils.GetMgmtServiceCluster(e.cluster.ID())
```

Comment:
`GetMgmtServiceCluster` internally call 2 API to get MC and SC, but we don't use SC here , is it good to have two func to get MC and SC ?

Reply:
ok, I think I can split them into two